src/pytom_tm/entry_points.py

```diff
@@ -1008,6 +1017,16 @@ def match_template(argv=None):
             )
         )
         per_tilt_weighting = True
+
+    elif args.warp_xml_file is not None:
+        voxel_size, tilt_angles, dose_accumulation, ctf_params, defocus_handedness = (
+            parse_warp_xml_data(
+                args.warp_xml_file,
+                args.tomogram,
+                phase_flip_correction=phase_flip_correction,
+            )
+        )
+        per_tilt_weighting = True
```

Comment:
I would not override the defocus handedness in the parse_warp_xml function, but instead set it to `args.defocus_handedness` inside this `if` statement (as is done on line 1037). This would keep the xml parser clean of random defaults that are not actually part of the xml, should default to `0` as that is the default for `args.defocus_handedness`, and still give a user the option to override the default if they absolutely want